boards/feather_esp32/definition.json

```diff
@@ -1,12 +1,25 @@
 {
-    "boardName":"adafruit-huzzah-32",
+    "boardName":"feather_esp32",
     "mcuName":"ESP32",
     "mcuRefVoltage": 3.3,
     "VID":"0x239A",
     "PID":"0x8036",
     "displayName":"Adafruit Feather HUZZAH ESP32",
     "productPageURL":"https://www.adafruit.com/product/3405",
     "documentationURL":"https://learn.adafruit.com/adafruit-huzzah32-esp32-feather",
+    "esptool": {
+      "fileSystemSize": "94208",
+      "blockSize": "4096",
+      "offset": "0x290000",
+      "structure": [
+        {
```

Comment:
this is an object nested inside an array, should just be an object, i believe (remove the wrapping `[]`)